### PR TITLE
P0972R0 <chrono> zero(), min(), and max() should be noexcept

### LINKED ISSUE #2415: [2018-11 LWG Motion 9] P0972R0 <chrono> zero(), min(), and max() should be noexcept
[P0972R0](http://wg21.link/p0972r0)

### PR DESCRIPTION
**Pull request by burblebee**
Fixes #2415.

### CODE REVIEW

File: source/time.tex [hunk 1]
```diff
@@ -1133,9 +1133,9 @@
 template<class Rep>
   struct duration_values {
   public:
-    static constexpr Rep zero();
-    static constexpr Rep min();
-    static constexpr Rep max();
+    static constexpr Rep zero() noexcept;;
```
Comment (aburch) [inline source/time.tex]: The extra semicolon should not be there?

(I have no experience with the C++ standardization process, so excuse me if this is the wrong way to point out a typo.)

Comment (burblebee) [reply source/time.tex]: Thanks for catching this!!

**Comment by burblebee** [comment]
Should I remove the "changes required" labels now that the typos have been fixed and pushed?

I assume an editor will assign themselves to this PR for a final review?

**Comment by jensmaurer** [comment]
@burblebee: Yes, you can remove the label after the fix. The label is just to call everybody's (including the author's) attention that this is not ready for final review.

**Pull request merged**